src/gui/search/searchtab.cpp

```diff
@@ -154,19 +174,131 @@ QStandardItemModel* SearchTab::getCurrentSearchListModel() const
+    filterModel->invalidate();
+    updateResultsCount();
+}
+
+void SearchTab::fillFilterComboBoxes()
+{
+    using namespace Utils::Misc;
+    QStringList unitStrings;
+    unitStrings.append(unitString(SizeUnit::Byte));
+    unitStrings.append(unitString(SizeUnit::KibiByte));
+    unitStrings.append(unitString(SizeUnit::MebiByte));
+    unitStrings.append(unitString(SizeUnit::GibiByte));
+    unitStrings.append(unitString(SizeUnit::TebiByte));
+    unitStrings.append(unitString(SizeUnit::PebiByte));
+    unitStrings.append(unitString(SizeUnit::ExbiByte));
+
+    m_ui->minSizeUnit->clear();
+    m_ui->maxSizeUnit->clear();
+    m_ui->minSizeUnit->addItems(unitStrings);
+    m_ui->maxSizeUnit->addItems(unitStrings);
+
+    m_ui->minSize->setValue(0);
+    m_ui->minSizeUnit->setCurrentIndex(static_cast<int>(SizeUnit::MebiByte));
+
+    m_ui->maxSize->setValue(-1);
+    m_ui->maxSizeUnit->setCurrentIndex(static_cast<int>(SizeUnit::TebiByte));
+
+    m_ui->filterMode->clear();
+
+    QMetaEnum nameFilteringModeEnum =
+        this->metaObject()->enumerator(this->metaObject()->indexOfEnumerator("NameFilteringMode"));
+
+    m_ui->filterMode->addItem(tr("Torrent names only"), nameFilteringModeEnum.valueToKey(OnlyNames));
+    m_ui->filterMode->addItem(tr("Everywhere"), nameFilteringModeEnum.valueToKey(Everywhere));
+
+    QVariant selectedMode = SettingsStorage::instance()->loadValue(QLatin1String(FILTER_MODE_SETTING_NAME),
+                                                                   nameFilteringModeEnum.valueToKey(OnlyNames));
+    int index = m_ui->filterMode->findData(selectedMode);
+    m_ui->filterMode->setCurrentIndex( index == -1 ? 0 : index);
+}
+
+QString SearchTab::statusText(SearchTab::Status st)
+{
+    switch (st) {
+    case Status::Ongoing:
+        return tr("Searching...");
+    case Status::Finished:
+        return tr("Search has finished");
+    case Status::Aborted:
+        return tr("Search aborted");
+    case Status::Error:
+        return tr("An error occurred during search...");
+    case Status::NoResults:
+        return tr("Search returned no results");
+    default:
+        return QString();
+    }
+}
+
+QString SearchTab::statusIconName(SearchTab::Status st)
+{
+    switch (st) {
+    case Status::Ongoing:
+        return QLatin1String("task-ongoing");
+    case Status::Finished:
+        return QLatin1String("task-complete");
+    case Status::Aborted:
+        return QLatin1String("task-reject");
+    case Status::Error:
+        return QLatin1String("task-attention");
+    case Status::NoResults:
+        return QLatin1String("task-attention");
+    default:
+        return QString();
+    }
+}
+
+void SearchTab::saveFilterModeAsDefault()
+{
+    SettingsStorage::instance()->storeValue(QLatin1String(FILTER_MODE_SETTING_NAME)
+                                            , m_ui->filterMode->itemData(m_ui->filterMode->currentIndex()));
```

Comment:
Fix coding style!
We starts new line with comma only in constructor initialization list.